Custom ScoreException

```diff
@@ -0,0 +1,54 @@
+
+public class Score{
+	String num;
+	String name;
+	String major;
+	int score;
+	String unit;
+	public String getName(){
+		return name;
+	}
+	public int getScore(){
+		return score;
+	}
+	public String getNum(){
+		return name;
+	}
+	public String major(){
+		return major;
+	}
+	Score(){
+		
+	}
+	Score(int score){
+		
+	}
+	public void score(int e) throws ScoreException{
+		if(e<0 || e>100){
+			throw new ScoreException(e);
+		}
+	}
+
+}
+public class ScoreException {
```

Comment:
ScoreException class should extend Throwable Class like:
class ScoreException extends Throwable{
....}